src/test/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSinkTest.java

```diff
@@ -23,7 +23,7 @@
 import static org.junit.Assert.*; // this is required by "fail" like assertions
```

Comment:
Not to be solved in this PR but shouldn't this class be named OrionHdfsSinkTest? :)